admin/database/database.go

```diff
@@ -59,7 +59,7 @@ type DB interface {
 	UpdateOrganizationAllUsergroup(ctx context.Context, orgID, groupID string) (*Organization, error)
 
 	FindOrganizationAutoinviteDomain(ctx context.Context, orgID string, domain string) (*OrganizationAutoinviteDomain, error)
-	FindOrganizationAutoinviteDomainsForOrganization(ctx context.Context, orgID string) ([]*OrganizationAutoinviteDomain, error)
+	FindOrganizationAutoinviteDomainsForOrganization(ctx context.Context, orgID string) ([]*OrganizationWhitelistedDomain, error)
```

Comment:
Feels redundant to have `OrganizationAutoinviteDomain` and `OrganizationWhitelistedDomain`. Let's keep it simple and resolve role IDs in the API handler.

Also, let's change `autoinvite` to `whitelisted` everywhere for consistency

Reply:
We are doing this at other places as well returning `Member` from `FindOrganizationMemberUsers` instead of `User` and `Invite` from `FindOrganizationInvites` instead of `OrganizationInvite` etc to prevent extra lookups and get all the information in one go using joins. Just pointing it out, will change it if you still feel so.

Sure will change `autoinvite` to `whitelisted`

Reply:
Yeah, I'm not entirely sure how we should approach joined types, but do feel our current `Invite` and `Member` types are also a bit misplaced (though at least `Member` doesn't clash with any other types, since we don't have `OrganizationMember` for example). 

In this particular case, the naming just ends up feeling too confusing (`OrganizationAutoinviteDomain` and `OrganizationWhitelistedDomain` sound like the exact same thing). Probably we should pick one of these options:

- Just resolve roles separately in the API. We could have a multi-finders like `FindOrganizationRolesByIDs(ctx context.Context, ids ...string) (map[string]*OrganizationRole, error)` to make it easier.
- Have clearly named expanded return types. So `OrganizationWhitelistedDomain` might mirror the actual table, then we could have `OrganizationWhitelistedDomainWithJoinedNames` to show the distinction. This would also have to be reflected in the `Find...` func names.